src/facts/purple-coneflower.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Purple Coneflower
+image:
+  src: ../images/purple-coneflower.jpg
+  url: https://www.flickr.com/photos/kewing/15264929397
+  license: CC BY-NC 2.0
+  creator: Keith Ewing
+  alt: Bee on Purple Coneflower
+source:
+  url: https://mnzoo.org/conservation/act-wildlife/plant-pollinators/
+  name: Minnesota Zoo
+category: habitat
+---
+Purple coneflowers are brightly colored and attract many pollinators.
```

Comment:
```suggestion
Brightly colored purple coneflowers attract many pollinators, and they are also native to Minnesota.
```

Reply:
I think this wording sounds better than before so I will change it to this for now.